pkg/api/common.go

```diff
@@ -92,14 +106,17 @@ type Config struct {
 	Auth         *Auth
 	MultiTenancy tenancy.Authorizer
 
+	PromscaleEnableFeatures     CommaSeparated
+	PromscaleEnabledFeaturesMap map[string]struct{}
```

Comment:
The idea behind https://github.com/timescale/promscale/pull/896#discussion_r754938768 was to unify those two vars into one by using parsing mechanism provided by `flag` package. It is confusing to have a list of elements in one variable and a map of the same elements in the other.